### DISCUSSION TITLE
v5.0.0 ready?

### DISCUSSION THREAD

**Discussion by ArthurSonzogni** [ArthurSonzogni/FTXUI#720] [General]
Hi everyone,

I was thinking releasing v5.0.0 soon. Is there anything that should be added/fixed before releasing it?

See the [CHANGELOG](https://github.com/ArthurSonzogni/FTXUI/blob/main/CHANGELOG.md)

**Comment by mavam** [comment]
LGTM as is.

You may want to tweak this changelog entry:

> Breaking: Ref becomes XxxOption in component constructors.

Seems like "Xxx" is yet to be replaced.

**Reply by ArthurSonzogni** [reply]
Thanks!
I updated it into:
```
- Breaking: `Ref<{Component}Option>` becomes `{Component}Option` in component constructors.
```
Where `{Component}Option` is one in this [file](https://github.com/ArthurSonzogni/FTXUI/blob/main/include/ftxui/component/component_options.hpp).
